### PR TITLE
#4007 Encourage all to use gradlew rather than gradle

### PR DESCRIPTION
**Pull request by chia7712**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4007)
I will test all dockerfiles locally 
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: gradlew [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+    ;;
+  MINGW* )
+    msys=true
+    ;;
+  NONSTOP* )
+    nonstop=true
+    ;;
+esac
+
+CLASSPATH=$APP_HOME/gradle/wrapper/gradle-wrapper.jar
```
Comment (vitojeng) [inline gradlew]: It seems we need append **gradle-wrapper.jar** ?

Comment (chia7712) [reply gradlew]: gradle-wrapper.jar  is binary file. The gradlew script download it automatically if gradle-wrapper.jar  does not exist.

Comment (vitojeng) [reply gradlew]: Where is the downalod script in gradlew script file? I cannot find it.

Comment (jackyoh) [reply gradlew]: 下載的位址在 gradle/wrapper/gradle-wrapper.properties 的 distributionUrl 變數裡

Comment (vitojeng) [reply gradlew]: yeah, you are right. But I think not contain any download script in the **gradlew** file.

Comment (chia7712) [reply gradlew]: seems the downlaod is included by gradle-wrapper.jar :(

let me add the binary to our source

**Comment by chia7712** [comment]
unrelated error

**Pull request merged**